tests/jerry/array-prototype-lastindexof.js

```diff
@@ -69,3 +69,50 @@ try {
   assert(e.message === "foo");
   assert(e instanceof ReferenceError);
 }
+
+// Checking behavior when there are no arguments except "this"
+try {
+  var a = "This is a sample text string to test this function";
+  Array.prototype.lastIndexOf.call(a);
```

Comment:
Add an `assert(false)` after this line, to ensure that an exception has been thrown.